### DISCUSSION TITLE
Durability Viewer

### DISCUSSION THREAD

**Discussion by AtlasOfEden** [BedrockTweaks/Files#368] [Ideas]
Would it be possible to add durability numbers similar to Java Edition through a resource pack?

**Comment by IdotIcom** [comment]
https://modbay.org/textures/1885-durability-tools-viewer.html

**Comment by SquatchHunter** [comment]
~~out of scope for this project, use the linked pack or find another alternative, do not think we will _ever_ really provide this sadly.~~
